lib/pure/httpclient.nim

```diff
@@ -1087,7 +1089,7 @@ proc request*(client: HttpClient | AsyncHttpClient, url: Uri | string,
       # The body is stripped away
       redirectBody = ""
       # Delete any header value associated with the body
-      if headers != nil:
+      if not headers.isNil():
```

Comment:
Why use `isNil`? AFAIR  `!= nil` style may be preferred.

Reply:
:shrug:

Reply:
https://discord.com/channels/371759389889003530/371759389889003532/696718991019999252